kernel/src/main/java/org/kframework/parser/json/JsonParser.java

```diff
@@ -155,8 +156,8 @@ public static FlatModule toFlatModule(JsonObject data) throws IOException {
         String name = data.getString("name");
 
         JsonArray jsonimports = data.getJsonArray("imports");
-        Set<String> imports = new HashSet<>();
-        jsonimports.getValuesAs(JsonString.class).forEach(i -> imports.add(i.getString()));
+        Set<Import> imports = new HashSet<>();
+        jsonimports.getValuesAs(JsonString.class).forEach(i -> imports.add(Import.apply(i.getString(), Att.empty())));
 
```

Comment:
Hmmmm, shouldn't we be correctly encoding/decoding the imports to JSON?

Reply:
As in, shouldn't we change the JSON encoding to include the attributes on imports instead of assuming it's empty?

Reply:
I mean, those attributes don't actually get kept. You are converting from a Module back to a FlatModule and the attributes will have been lost by that point anyway. We would need a much larger change that tracked the attributes of imports through the entire compilation process if we were going to be able to do what you suggest here. But since imports can't actually have attributes and the attributes are just placeholders for the location, and the location information isn't really needed by the json, I figured this was good enough.